microservices/submission-service/src/replay-validation/replay-validation.service.ts

```diff
@@ -109,80 +106,61 @@ export class ReplayValidationService {
             };
         }
 
+        const gameResult = await this.coreService.send(CoreEndpoint.GetGameByGameMode, {gameModeId: scrim.gameMode.id});
+        if (gameResult.status === ResponseStatus.ERROR) {
+            this.logger.error(`Unable to get game gameMode=${scrim.gameMode.id}`);
+            return {
+                valid: false,
+                errors: [ {error: `Failed to find associated game. Please contact support.`} ],
+            };
+        }
+
         // Get platformIds from stats
-        const ballchasingIds = stats.flatMap(s => [s.blue, s.orange].flatMap(t => t.players.flatMap(p => ({
-            platform: p.id.platform,
-            id: p.id.id,
+        // Don't send the same request for multiple players
+        const uniqueBallchasingPlayerIds = Array.from(new Set(stats.flatMap(s => [s.blue, s.orange].flatMap(t => t.players.flatMap(p => ({
             name: p.name,
```

Comment:
`name` is unused, can we remove it?

Reply:
It is used later on to return a list of invalid players. 143